core/scripts/vrfv2/testnet/super_scripts.go

```diff
@@ -23,7 +23,7 @@ func deployUniverse(e helpers.Environment) {
 	// required flags
 	linkAddress := deployCmd.String("link-address", "", "address of link token")
 	linkEthAddress := deployCmd.String("link-eth-feed", "", "address of link eth feed")
-	subscriptionBalanceString := deployCmd.String("subscription-balance", "", "amount to fund subscription")
+	subscriptionBalanceString := deployCmd.String("subscription-balance", "10000000000000000000", "amount to fund subscription")
 
 	// optional flags
 	fallbackWeiPerUnitLink := deployCmd.String("fallback-wei-per-unit-link", "60000000000000000", "fallback wei/link ratio")
```

Comment:
Can do something similar here, but can use the appropriate helper for the denomination. i.e if it's 1000 GWei, can do `assets.GWei(1000).String()`.